beesdoo_product/models/beesdoo_product.py

```diff
@@ -64,12 +64,20 @@ class BeesdooProduct(models.Model):
         translate=True,
     )
 
-    main_seller_id = fields.Many2one(
-        "res.partner",
-        string="Main Seller",
+    top_supplierinfo_id = fields.Many2one(
+        comodel_name="product.supplierinfo",
```

Comment:
Why not `main_supplierinfo_id` ? It would be more consistent no ? Is there a reason for using "top" ?

Reply:
Because this field is a technical field, and it's not the main supplier that is selected but the first one. So top_supplier reflect more the reality.

Reply:
OK :+1: